gematria/datasets/pipelines/benchmark_cpu_scheduler.py

```diff
@@ -17,6 +17,7 @@
 from collections.abc import Iterable
 import os
 import re
+from enum import Enum
```

Comment:
The no-direct-imports rule applies here (only `collections.abc`, `typing`, and `typing_extensions` have an exception).